### PR TITLE
Update grid patterns on index, /about, /blog, /community, /contact-us and /core pages

### LINKED ISSUE #5645: Update grid patterns pt. 1
Per point 2 of https://ubuntu.com/blog/vanilla-framework-2-0-upgrade-guide

Update grid row/column markup on homepage, /about, /blog, /community, /contact-us, /containers and /core pages

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- updated grid markup on the following pages (and their descendants) to ensure that `.col-` elements are the only immediate children of `.row` elements:
  - /about
  - /blog
  - /community
  - /contact-us
  - /core

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check layout of main content on these pages is not broken (ignoring site-wide components like navigation and footer):
  - /about
    - /about/packages
    - /about/release-cycle
  - /blog
    - /blog/topics/design (or any topic page)
    - /blog/cloud-and-server
    - /blog/desktop
    - /blog/internet-of-things
    - /blog/press-centre
    - /blog/archives
    - /blog/(any post)
  - /community
    - /community/canonical
    - /community/code-of-conduct
    - /community/debian
    - /community/diversity
    - /community/governance
    - /community/mission
  - /contact-us
    - /contact-us/thank-you
  - /containers
  - /core
  - /core/thank-you

## Issue / Card

Fixes #5645

### CODE REVIEW

File: templates/about/release-cycle.html [hunk 1]
```diff
@@ -229,12 +247,15 @@ <h2>Maintenance and security updates</h2>
...
-    <p>In addition, the kernel versions from the subsequent four releases are made available on the latest LTS release of Ubuntu. So Ubuntu 16.04 LTS received the kernels from Ubuntu 16.10, 17.04, 17.10 and 18.04 LTS. These kernels use newer upstream versions and as a result offer an easy path to newer features and newer classes of hardware for many users of Ubuntu. Note however that these kernels ‘roll’ which means that they jump every six months until the next LTS. Large scale deployments that adopt these ‘hardware enablement’ or HWE kernels should manage those transitions explicitly. These newer HWE kernels are accompanied by a collection of userspace tools closely tied to the kernel and hardware, specifically X display enablement on newer graphics cards.</p>
-    <p>The Ubuntu kernel support lifecycle is as follows:</p>
+    <div class="col-12">
```
Comment (Lukewh) [inline templates/about/release-cycle.html]: `u-fexed-wedth`

File: templates/about/release-cycle.html [hunk 2]
```diff
@@ -366,8 +387,11 @@ <h2>Ubuntu kernel release cycle</h2>
       </table>
     </div>
   </div>
+
   <div class="row">
-    <p>For more information on previous and upcoming kernel releases please see the <a href="https://wiki.ubuntu.com/Kernel/LTSEnablementStack " class="p-link--external ">Ubuntu LTS Enablement Stack wiki page</a>.</p>
+    <div class="col-12">
```
Comment (Lukewh) [inline templates/about/release-cycle.html]: `me-fluid-breadth`

File: templates/about/release-cycle.html [hunk 3]
```diff
@@ -554,90 +579,94 @@ <h2>Ubuntu OpenStack release cycle</h2>
       </table>
     </div>
   </div>
+
   <div class="row">
-    <p>For more information on previous and upcoming Ubuntu OpenStack releases please see the <a href="https://wiki.ubuntu.com/OpenStack/CloudArchive " class="p-link--external ">Ubuntu Cloud Archive wiki page</a>.</p
+    <div class="col-12">
```
Comment (Lukewh) [inline templates/about/release-cycle.html]: `u-get-the-point`

File: templates/about/release-cycle.html [hunk 4]
```diff
@@ -554,90 +579,94 @@ <h2>Ubuntu OpenStack release cycle</h2>
...
-        <p>The release cycle of the Charmed Distribution of Kubernetes (CDK) is tightly
-          synchronised to the release of upstream Kubernetes<sup>&reg;</sup>.
-          The current release and two prior releases are supported, giving an effective
-          support period of nine months, subject to changes in the upstream release
-          cycle.</p>
-          <p>The CDK support lifecycle can be represented this way:</p>
-        </div>
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
```
Comment (Lukewh) [inline templates/about/release-cycle.html]: `u-shouldn't change this, it's correct to be a row with a col-8`

File: templates/about/release-cycle.html [hunk 5]
```diff
@@ -554,90 +579,94 @@ <h2>Ubuntu OpenStack release cycle</h2>
...
+              <td>September 2019</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+  
+  <div class="row">
+    <div class="col-12">
```
Comment (Lukewh) [inline templates/about/release-cycle.html]: `u-know`

File: templates/community/code-of-conduct.html [hunk 6]
```diff
@@ -13,7 +13,7 @@ <h1>Ubuntu Code of Conduct v2.0</h1>
     </div>
   </div>
   <div class="row">
-    <div class="u-measure--p">
+    <div class="col-12">
```
Comment (Lukewh) [inline templates/community/code-of-conduct.html]: last one I'm going to call out :D

File: templates/about/release-cycle.html [hunk 7]
```diff
@@ -9,31 +9,34 @@
...
+      <img src="{{ ASSET_SERVER_URL }}ed348358-logo-cof.svg" width="200" alt="ubuntu logo">
     </div>
   </div>
 </section>
 
 <section class="p-strip--light is-bordered">
   <div class="row">
-    <h2>Version numbers are YY.MM</h2>
-    <p>Releases of Ubuntu get a development codename (‘Breezy Badger’) and are versioned by the year and month of delivery - for example Ubuntu 17.10 was released in October 2017.</p>
+    <div class="col-12">
```
[the remark this answers is not in the document]
Comment (sowasred2012) [reply templates/about/release-cycle.html]: Nice - missed that point! Have updated all of these bad boys

**Pull request merged**